src/components/input/Button.tsx

```diff
@@ -1,6 +1,17 @@
-export enum BUTTON_VARIANTS {}
-// TODO: Add button variants
-// ex PRIMARY = "primary classname",
+export enum BUTTON_VARIANTS {
+  PRIMARY_LG = "bg-blue-500 text-white py-3 px-4 text-title-xl hover:shadow-blue-700 hover:bg-blue-600 active:bg-blue-500",
+  PRIMARY_SM = "bg-blue-500 text-white py-2.5 px-4 text-title-lg hover:shadow-blue-700 hover:bg-blue-600 active:bg-blue-500",
+  DANGER_LG = "bg-danger-200 text-white py-3 px-4 text-title-xl hover:shadow-danger-400 hover:bg-danger-300 active:bg-danger-200",
+  DANGER_SM = "bg-danger-200 text-white py-2.5 px-4 text-title-lg hover:shadow-danger-400 hover:bg-danger-300 active:bg-danger-200",
+  SUCCESS_LG = "bg-success-200 text-white py-3 px-4 text-title-xl hover:shadow-success-400 hover:bg-success-300 active:bg-success-200",
+  SUCCESS_SM = "bg-success-200 text-white py-2.5 px-4 text-title-lg hover:shadow-success-400 hover:bg-success-300 active:bg-success-200",
+  WARNING_LG = "bg-warning-200 text-black py-3 px-4 text-title-xl hover:shadow-warning-400 hover:bg-warning-300 active:bg-warning-200",
+  WARNING_SM = "bg-warning-200 text-white py-2.5 px-4 text-title-lg hover:shadow-warning-400 hover:bg-warning-300 active:bg-warning-200",
```

Comment:
all of these reuse px-4 is it possible to combine these classes some how using conditional rendering?

Reply:
Maybe you can even break them down further into SIZE variants which deal with whether something is large or small, and keep BUTTON_VARIENTS for color schemes